src/client/dfs/SConscript

```diff
@@ -19,12 +18,13 @@ def scons():
     libraries = libraries + ['dfs']
 
     denv.AppendUnique(LIBPATH=[Dir('.')])
-    dfuse_hl_libs = libraries + ['fuse3']
 
+    fenv = denv.Clone()
+    prereqs.require(fenv, 'fuse')    
     dfuse_hl_src = ['dfuse_hl.c']
-    dfuse_hl = daos_build.program(denv, 'dfuse_hl', dfuse_hl_src,
-                                  LIBS=dfuse_hl_libs)
-    denv.Install('$PREFIX/bin/', dfuse_hl)
+    dfuse_hl = daos_build.program(fenv, 'dfuse_hl', dfuse_hl_src,
+                                  LIBS=libraries)
```

Comment:
This is an example of where the old code was wrong, if you pass in LIBS like this then it replaces existing values with the new ones, so the prereqs.require() like will have added -lfuse, then the LIBS setting overwrites it which is why it had to be specifically added back into LIBS, the fix for that is to add $LIBS to the array as I did on L12.